swarm/pss/api.go

```diff
@@ -177,3 +186,7 @@ func (pssapi *API) GetPeerTopics(pubkeyhex string) ([]Topic, error) {
 func (pssapi *API) GetPeerAddress(pubkeyhex string, topic Topic) (PssAddress, error) {
 	return pssapi.Pss.getPeerAddress(pubkeyhex, topic)
 }
+
+func checkMsg(msg []byte) bool {
```

Comment:
No need to split `checkMsg` and the actual error message. A `validate` method that returns the error is better.